content/webapp/pages/opening-times.js

```diff
@@ -78,37 +84,41 @@ export class OpeningTimesPage extends Component<Props> {
         }
         Body={<Body body={[]} />}
       >
-
+        {showNewOpeningHours &&
+          <OpeningTimesStatic />
```

Comment:
Bike shedding, but you could just have in the render function:
```
render() {
  if (showNewOpeningHouts) {
    return <OpeningTimesStatic />
  }
  // ... get on with business as usual
}
```

Just stops all the white space changes.
I only mention it as it introduces a toggle with as little change, and thus things to review and be worried about.

But as I said, bikeshedding and not worried.